config/bolt/config.yaml

```diff
@@ -199,3 +199,11 @@ user_avatar:
     upload_path: avatars
     extensions_allowed: ['png', 'jpeg', 'jpg', 'gif']
     default_avatar: '' # Put path of file locate in files directory
+
+# Settings for the reset password logic
+reset_password_settings:
+    show_already_requested_password_notice: true
+    mail_from: "do-not-reply@dummy.com"
```

Comment:
I think you shouldn't use a 'fake' domain that exists and isn't explicitly documented as usable in these kind of configurations (e.g. https://en.wikipedia.org/wiki/Example.com), in case someone doesn't change this mail bounces could actually be delivered to this address and might include sensitive data. You could use example.com/example.org but that is more intended for documentation, maybe do-not-reply@localhost or do-not-reply@invalid is a better option)

Reply:
didn't know that, good to know. bob, can you handle that please since you created that PR from the internal branch?

Reply:
otherwise you can leave it as is and I will create a PR after this has been merged

Reply:
I've changed it to `do-not-reply@example.org`